### PR TITLE
OSDOCS#7610: Update MicroShift sos report docs

### PR DESCRIPTION
**Pull request by kelbrown20**
**Version(s):** 4.14+
**Issue:** [OSDOCS-7610](https://issues.redhat.com//browse/OSDOCS-7610)

**Link to docs preview:**

[MicroShift sos report -> Gathering data from a Red Hat build of MicroShift sos report](https://66746--docspreview.netlify.app/microshift/latest/microshift_support/microshift-sos-report#gathering-data-microshift-sos-report_microshift-sos-report)

**QE review:**
- [x] QE has approved this change.

### CODE REVIEW

File: modules/microshift-gathering-sos-report.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@
 +
 [source,terminal]
 ----
-$ sos report --batch --clean --all-logs --profile microshift
+$ sos report --batch --all-logs --profile microshift
```
Comment (dhensel-rh) [inline modules/microshift-gathering-sos-report.adoc:21]: Should this documentation be using `/usr/bin/microshift-sos-report` to collect SOS logs  ?

Comment (kelbrown20) [reply modules/microshift-gathering-sos-report.adoc:21]: @copejon WDYT for this?

**Pull request merged**